python/pyarrow/_compute.pyx

```diff
@@ -901,10 +903,32 @@ class RoundTemporalOptions(_RoundTemporalOptions):
         "nanosecond".
     week_starts_monday : bool, default True
         If True, weeks start on Monday; if False, on Sunday.
-    """
-
-    def __init__(self, multiple=1, unit="day", week_starts_monday=True):
-        self._set_options(multiple, unit, week_starts_monday)
+    ceil_is_strictly_greater : bool, default False
+        If True times exactly on unit multiple boundary will be rounded
+        up one unit. This applies for ceiling only.
```

Comment:
Can you update this docstring as well? (following the update in C++)

Reply:
Done.